### DISCUSSION TITLE
Cannot use path@version syntax in GOPATH mode

### DISCUSSION THREAD

**Discussion by 077uyp** [projectdiscovery/nuclei#1427] [Q&A] [answered]
can't load package: package github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest: cannot use path@version syntax in GOPATH mode

**Comment by ehsandeep** [comment, marked as answer]
@077uyp that's the issue with your go environment, make sure you have **go1.17+**  installed on your system before installing nuclei.
